app/src/main/java/org/sopt/certi/core/component/button/BasicButton.kt

```diff
@@ -0,0 +1,119 @@
+package org.sopt.certi.core.component.button
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberUpdatedState
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.certi.core.util.pressedClickable
+import org.sopt.certi.ui.theme.Gray100
+import org.sopt.certi.ui.theme.Gray400
+import org.sopt.certi.ui.theme.LocalCertiTypographyProvider
+import org.sopt.certi.ui.theme.MainBlue
+import org.sopt.certi.ui.theme.PurpleBlue
+import org.sopt.certi.ui.theme.White
+
+@Composable
+fun CertiBasicButton(
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    buttonText: String = "",
+    onClick: () -> Unit = {}
+) {
+    val typoProvider = LocalCertiTypographyProvider.current
```

Comment:
[1]: 디자인 시스템을 사용하는 방식으로 수정해주세요 !
우리 이미 디자인 시스템을 CertiTheme 안에 CompositionLocalProvider를 통해 잘 구성해두었기 때문에, 이렇게 쓰시면 중복 선언이 발생합니더
저희 이쁘게 세팅 해둔 걸 사용해봐요 ~~~~!!

```suggestion
import org.sopt.certi.ui.theme.CertiTheme

    Text(
        text = "zzz",
        style = CertiTheme.typography.body.bold_16
    )
```
요런 식으로 쓰시면 될 것 같네요잉